examples/BasicExample/ESP32BasicExample.ino

```diff
@@ -0,0 +1,133 @@
+#include <WiFi.h>
+#include <SocketIoClient.h>
+
+/////////////////////////////////////
+////// USER DEFINED VARIABLES //////
+///////////////////////////////////
+/// WIFI Settings ///
+const char* ssid     = "yourSSID";
+const char* password = "yourWiFiPassword";
+
+/// Socket.IO Settings ///
+char host[] = "192.168.1.2"; // Socket.IO Server Address
+int port = 5000; // Socket.IO Port Address
+char path[] = "/socket.io/?transport=websocket"; // Socket.IO Base Path
```

Comment:
/socket.io/?transport=websocket, I have put the same path in the code in my system it works but if I try to put a different path along with the host and port that I hosted but it is getting disconnected why? and what is /socket.io/?transport=websocket"; // Socket.IO Base Path  mean